tidb-cloud/changefeed-sink-to-mysql.md

```diff
@@ -78,14 +74,40 @@ The **Sink to MySQL** connector can only sink incremental data from your TiDB cl
 
 After completing the prerequisites, you can sink your data to MySQL.
 
-1. Navigate to the **Changefeed** tab of your TiDB cluster.
+1. Navigate to the **Data Replication** > **Changefeed** tab of your TiDB cluster.
+
 2. Click **Sink to MySQL**.
-3. Fill in the MySQL URL, user, and password.
-    - If you already have data in your TiDB Cluster, you must fill in a specific TSO number that Dumpling provides.
-    - If you do not have any data in your TiDB Cluster, you can choose the "current" TSO.
-4. Click **Test Connectivity**. If your TiDB Cluster can connect to the MySQL service, the **Confirm** button is displayed.
-5. Click **Confirm** and after a while, the sink will begin its work, and the status of the sink will be changed to "**Producing**".
-6. After the operation is complete, set the GC time back (the default value is `10m`):
+
+3. Fill in the MySQL Endpoints, user name, and password in **MySQL Connection**.
+
+4. Click **Next** to test whether TiDB connects to MySQL successfully:
+
+    - If yes, you are directed to the next step of configuration.
+    - If not, the connectivity error is displayed, and you need to handle the error. After the error is resolved, click **Next** again.
+
+5. Customize **Table Filter** to filter the tables that you want to replicate. For the rule syntax, refer to [table filter rules](/table-filter.md)
+
+    - **Add filter rules**: you can set filter rules in this column. By default, there is a rule `*. *`, which stands for replicating all tables. When you add a new rule, TiDB Cloud queries all the tables in TiDB and displays the tables that can be replicated in the list box on the right. Instructions for use:
+    - **Tables to be replicated**: this column shows the tables to be replicated. But it does not show the new tables to be replicated in the future or the schema to be fully replicated.
+    - **Tables without valid keys**: this column shows tables that do not have unique and primary keys. Because no unique identifier can be used by the downstream system to handle duplicate events, the data might be inconsistent during replication. We recommend that you add unique keys or primary keys to these tables, or set filter rules, such as filtering out the table `test`.`tbl1` using "!test.tbl1".
```

Comment:
```suggestion
    - **Tables without valid keys**: this column shows tables without unique and primary keys. For these tables, because no unique identifier can be used by the downstream system to handle duplicate events, their data might be inconsistent during replication. To avoid such issues, it is recommended that you add unique keys or primary keys to these tables before the replication, or set filter rules to filter out these tables. For example, you can filter out the table `test.tbl1` using "!test.tbl1".
```